docs/new/docs/policy-language.md

```diff
@@ -323,106 +236,97 @@ By defining composite values in terms of variables and references, rules can def
 
 ### Objects
 
 Objects are unordered key-value collections. In Rego, any value type can be
 used as an object key. For example, the following assignment maps port **numbers**
 to a list of IP addresses (represented as strings).
 
-```live:eg/objects:module:merge_down
+```rego
+package objects
+
 ips_by_port := {
-    80: ["1.1.1.1", "1.1.1.2"],
-    443: ["2.2.2.1"],
+    80: ["10.0.0.1", "10.10.10.1"],
+    443: ["10.1.1.1"],
 }
-```
 
-```live:eg/objects/lookup:query:merge_down
-ips_by_port[80]
+result := ips_by_port[80]
 ```
 
-```live:eg/objects/lookup:output:merge_down
-```
-
-```live:eg/objects/iteration:query:merge_down
-some port; ips_by_port[port][_] == "2.2.2.1"
-```
-
-```live:eg/objects/iteration:output
-```
+<RunSnippet id="objects.rego" command="data.objects.result"/>
 
 When Rego values are converted to JSON non-string object keys are marshalled
 as strings (because JSON does not support non-string object keys).
 
-```live:eg/objects/marshal:query:merge_down
-ips_by_port
-```
+```rego
+package objects
 
-```live:eg/objects/marshal:output
+# when queried, this will be converted to JSON
+json := ips_by_port
 ```
 
+<RunSnippet files="#objects.rego" command="data.objects.json"/>
+
 ### Sets
 
 In addition to arrays and objects, Rego supports set values. Sets are unordered
 collections of unique values. Just like other composite values, sets can be
 defined in terms of scalars, variables, references, and other composite values.
 For example:
 
-```live:eg/cuboid/sets:query:merge_down
-s := {cuboid.width, cuboid.height, cuboid.depth}
-```
-
-```live:eg/cuboid/sets:output
-```
-
-> Set documents are collections of values without keys. OPA represents set
-> documents as arrays when serializing to JSON or other formats that do not
-> support a set data type. The important distinction between sets and arrays or
-> objects is that sets are unkeyed while arrays and objects are keyed, i.e., you
-> cannot refer to the index of an element within a set.
+```rego
+package sets
 
-When comparing sets, the order of elements does not matter:
+s1 := {1,2,3}
+s2 := {3,2,1}
 
-```live:eg/set_equality:query:merge_down
-{1,2,3} == {3,1,2}
+sets_equal := s1 == s2
 ```
 
-```live:eg/set_equality:output
-```
+<RunSnippet command="data.sets"/>
 
-Because sets are unordered, variables inside sets must be unified with a ground
-value outside of the set. If the variable is not unified with a ground value
-outside the set, OPA will complain:
+:::warning
+Set documents are collections of values without keys or order. OPA represents
+sets as arrays when serializing to JSON or other formats that do not support a
+set data type. The important distinction between sets and arrays or objects is
+that sets are unkeyed while arrays and objects are keyed, i.e., you cannot refer
+to the index of an element within a set.
+:::
 
-```live:eg/set_unification:query:merge_down
-{1,2,3} == {3,x,2}
-```
+<RunSnippet command="data.sets.result"/>
```

Comment:
<img width="677" alt="image" src="https://github.com/user-attachments/assets/05fad0c4-ace5-4ffe-8dc0-f0deac70af11" />

Reply:
Ahh, that should not be there bb541a5226a9ef3c35d5cf8119e1adb78ae8cd05